nvi-commons/src/test/java/no/sikt/nva/nvi/common/service/NviServiceTest.java

```diff
@@ -100,9 +101,10 @@ void shouldCreateAndFetchPublicationById() {
         var createdCandidate = nviService.upsertCandidate(candidate).orElseThrow();
         var createdCandidateId = createdCandidate.identifier();
 
-        var fetchedCandidate = nviService.findCandidateById(createdCandidateId).orElseThrow().candidate();
+        var fetchedCandidate = nviService.findCandidateById(createdCandidateId).orElseThrow();
 
-        assertThat(fetchedCandidate, is(equalTo(candidate)));
+        assertThat(fetchedCandidate.periodStatus().startDate(), is(not(nullValue())));
```

Comment:
Not sure what you are testing here, since you are asserting that `fetchedCandidate.candidate()` is equal to (expected) `candidate` in the next line. If you want to test something specific for PeriodStatus, create a new test and assert if `fetchedCandidate.periodStatus().startDate()`  or `fetchedCandidate.periodStatus()` is equal to expected value?